spec/models/like_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject do
+    Like.create do |like|
+      like.author = User.create do |user|
+        user.name = 'John Doe'
+        user.photo = 'https://picsum.photos/200'
+        user.bio = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+      end
```

Comment:
- [ ] The same here. So far great job 👏🏻 However,  you are missing a test to test the `update_likes_counter` method of your Like model, please add this ✍🏻

```ruby
  def update_likes_counter
    post.increment!(:likes_counter)
  end
```